static/app/views/replays/detail/console/index.tsx

```diff
@@ -31,6 +34,20 @@ function Console({breadcrumbs, startTimestamp = 0}: Props) {
     [logLevel, searchTerm, breadcrumbs]
   );
 
+  const maxActiveConsoleIndex = useMemo(() => {
```

Comment:
I think we want this to work similarly to the breadcrumbs where we highlight the closest message(s).

Reply:
Ah okay so like highlight the chunk of items that occur within the same timespan but don't go backwards to highlight everything that came before.

Reply:
https://user-images.githubusercontent.com/7014871/170291923-07cfaa7a-a7c9-4ed5-b312-7dd6b601796a.mov